### ISSUE TITLE
Improve testing to check for edge cases

### ISSUE THREAD

**Issue by anshuldutt21** [spdx/spdx-online-tools#153]
Some tests need to done to ensure proper workflow of the the website. Refer to #149 for example.

**Comment by jlovejoy** [comment]
I think this have been addressed via the other referenced issue, so we can close? @goneall

**Comment by goneall** [comment]
@jlovejoy It's a pretty general issue and still applicable to other workflow items.  I'm 50/50 on keeping it open.

@rtgdk what do you think?

**Comment by rtgdk** [comment]
Yeah, the automated testing state is in very bas shape right now. Some tests don't even finish locally due to jpypy getting stuck. Also, tests hardly covers all the cases so I think we should keep this one open.
I plan to take a look at this once I get time 😅

**Comment by bact** [comment]
The test workflow is in a better shape now. It runs successfully for every pull request. I will close this.

Please reopen if we have a specific edge case to test in mind.
